packages/hackmcx-api/package.json

```diff
@@ -9,11 +9,12 @@
     "api-seed": "node ./src/index.js --seed"
   },
   "dependencies": {
+    "bcrypt": "^5.0.1",
+    "cors": "^2.8.5",
```

Comment:
It seems like the only thing changing here is the addition of bcrypt. We might want to consider removing the other changes, even it they do reorganize the imports in alphabetical order.